flutter-studio/resources/META-INF/studio-contribs.xml

```diff
@@ -0,0 +1,18 @@
+<!-- Defines Android Studio IDE-specific contributions and implementations. -->
+<idea-plugin>
+
+  <extensions defaultExtensionNs="com.android">
+    <moduleDescriptionProvider implementation="io.flutter.module.FlutterDescriptionProvider"/>
+  </extensions>
+
+  <actions>
+
+    <action id="flutter.NewProject" class="io.flutter.actions.FlutterNewProjectAction"
+            text="New Flutter Project..."
```

Comment:
That would be inconsistent with AS and IDEA. I just did a quick (and possibly incomplete) search and the only menu items I found that use the ellipsis character are in the Flutter plugin :)